app/templates/components/file_upload.html

```diff
@@ -0,0 +1,74 @@
+{% macro js(file_field_id, file_url_id, preview_id) %}
+
+<script>
+
+if (!window.getSignedRequest) {
```

Comment:
My initial thought was that this is just an example so wanted to keep it all in the html with the example. On second thought, we'll need something non-react for admin, so I'll do a jquery version in its own js file.